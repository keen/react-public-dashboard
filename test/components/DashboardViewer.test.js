import React from 'react'
import { shallow } from 'enzyme';
import Viewer from '../../lib/DashboardViewer';

it('Viewer component renders correctly', () => {
  const props = {
    dashboardId: '5cf51f130d27b300169b7f27',
    projectId: '5c87b64ec9e77c0001cf5b6e',
    accessKey: 'AB09386091AC7DE2B9E8354B6DD633442955A98B11A41FBD0AFF33A1CE1021A2'
  };
  const wrapper = shallow(<Viewer {...props} />);
  expect(wrapper).toMatchSnapshot();
});