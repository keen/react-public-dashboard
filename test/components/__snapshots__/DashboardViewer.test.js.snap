// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Viewer component renders correctly 1`] = `ShallowWrapper {}`;
